arm-web/2016-09-01/swagger/AppServiceEnvironments.json

```diff
@@ -4197,6 +4414,67 @@
     "WorkerPool": {
```

Comment:
Yes this is intentional. Same explanation as AppServiceCertificate vs AppServiceCertificateResource. Both models are used.